db/seeds.rb

```diff
@@ -1,17 +1,27 @@
+require 'random_data'
+
 User.destroy_all
 
-User.create!(
+user1 = User.create!(
   name: 'user1',
   email: 'user1@example.com',
   password: 'hello123',
+  password_confirmation: 'hello123'
 )
 
-User.create!(
+user1.skip_confirmation!
+user1.save
+
+user2 = User.create!(
   name: "user2",
   email: "user2@example.com",
-  password: "hello123"
+  password: "hello123",
+  password_confirmation: 'hello123'
 )
 
+user2.skip_confirmation!
+user2.save
+
```

Comment:
In `seeds.rb`, lines 5 to 13 are very similar to lines 15 to 23.  How could these lines be rewritten to remove the duplication?